nixos/community.tt

```diff
@@ -1,225 +1,91 @@
-    nixos</strong>).
-</p>
-
-<h3>Documentation</h3>
-
-<ul>
-
-  <li>The <a href="[%nixosManual%]">NixOS manual</a> has a <a
-  href="[%nixosManual%]#ch-development">whole chapter on NixOS
-  hacking</a>. Also, check out the sections on <a
-  href="[%nixosManual%]#sec-configuration-syntax">the syntax of NixOS
-  modules</a> and on <a href="[%nixosManual%]#sec-custom-packages">how
-  to add your own packages</a>.</li>
-
-  <li>The <a href="[%nixpkgsManual%]">Nixpkgs manual</a> has a lot of
-  information on how to add packages to Nixpkgs.  In particular, you
-  should check out the <a
-  href="[%nixpkgsManual%]#chap-quick-start">“Quick Start” chapter</a>
-  for an overview of the process of adding a package.</li>
-
-  <li>The Nix manual has <a
-  href="[%nixManual%]#chap-writing-nix-expressions">a chapter on
-  writing Nix expressions</a>, including the syntax and semantic of
-  the Nix expression language.</li>
-
-  <li>There is a <a
-  href="https://nixos.org/~eelco/pubs/nixos-jfp-submitted.pdf">paper
-  on NixOS</a> has a lot of information on the NixOS module
-  system (especially section 6).</li>
-
-</ul>
-
-<h3>Source repositories</h3>
-
-<p>The sources of all Nix-related projects are in the <a
-href="https://github.com/NixOS/">NixOS organization on
-GitHub</a>. NixOS lives in the <a
-href="https://github.com/NixOS/nixpkgs/tree/master/nixos">nixos</a>
-subdirectory of the Nixpkgs repository.</p>
-
-<h3>Discourse</h3>
-
-<p><a href="https://discourse.nixos.org">discourse.nixos.org</a> hosts
-a community of Nix developers and users, providing resources to help
-you:</p>
-
-<ul>
-<li>find answers to your Nix/Nixpkgs/NixOS/NixOps questions</li>
-<li>discuss current and future directions of the project</li>
-<li>evangelise and promote Nix in your community</li>
-</ul>
-
-<h3>IRC channel</h3>
-
-<p>The NixOS developers hang out on the <a
-href="irc://irc.freenode.net/#nixos"><tt>#nixos</tt> channel</a> on <a
-href="https://freenode.net/"><tt>irc.freenode.net</tt></a>.  This
-channel is <a href="https://logs.nix.samueldr.com/nixos/">logged</a>.</p>
-
-<h3>Blogs</h3>
-
-<p><a href="https://planet.nixos.org/">Planet</a> aggregates blog posts written
-by the community and <a href="https://www.reddit.com/r/NixOS/">/r/NixOS</a>
-also contains useful resources.</p>
+<h1>NixOS Community</h1>
 
-<p>To add yourself to <a href="https://planet.nixos.org/">Planet NixOS</a>,
-fork the <a
-href="https://github.com/NixOS/nixos-org-configurations">nixos-org-configurations</a>
-repo and add the feed source to <a
-href="https://github.com/NixOS/nixos-org-configurations/blob/master/nixos-org/planet-feeds.nix">planet-feeds.nix</a>
-then make a pull request.</p>
+<p class="lead">Nix and NixOS are developed and used by a diverse and welcoming community from all around the world.</p>
 
-<h3>Meetups</h3>
+<p><strong>Check out what the community has made recently in the <a href="https://weekly.nixos.org/">NixOS Weekly</a> newsletter.</strong></p>
```

Comment:
I wanted to make this more prominently. Now i found a way by making it **bold** :)